src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,182 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static final int MAX_NUMBER_OF_TASK = 100;
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+        String command;
+        Task[] tasks = new Task[MAX_NUMBER_OF_TASK];
```

Comment:
I like how you place the array specifier immediately after Task to indicate that the type of the variable is an array of Task type elements.